guardian/workflow-frontend
# Bump AngularJS to 1.8.3

## What does this change?

Bumps AngularJS and related dependencies to 1.8.3.

Borrows from https://github.com/guardian/flexible-content/pull/4346.

### How to test

There's very little test coverage of the UI components – just nine tests for a date parser, written nine years ago 😅 

As a result, this PR is worth a thorough test in CODE to be confident that users won't see problems.

Recommend testing with dev tools open, as errors caused by this upgrade are likely to result in red message of the sort mentioned above:  `ctrl.{propertyName} is undefined`.
